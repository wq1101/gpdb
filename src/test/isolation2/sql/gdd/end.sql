-- disable GDD

-- start_ignore
! gpconfig -c gp_enable_global_deadlock_detector -v off;
! gpstop -rai;
-- end_ignore

1: SHOW gp_enable_global_deadlock_detector;
