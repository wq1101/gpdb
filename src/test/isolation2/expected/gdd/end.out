-- disable GDD

-- start_ignore
! gpconfig -c gp_enable_global_deadlock_detector -v off;
20200424:15:51:08:052278 gpconfig:zlv:gpadmin-[INFO]:-completed successfully with parameters '-c gp_enable_global_deadlock_detector -v off'

! gpstop -rai;
20200424:15:51:08:052977 gpstop:zlv:gpadmin-[INFO]:-Starting gpstop with args: -rai
20200424:15:51:08:052977 gpstop:zlv:gpadmin-[INFO]:-Gathering information and validating the environment...
20200424:15:51:08:052977 gpstop:zlv:gpadmin-[INFO]:-Obtaining Greenplum Master catalog information
20200424:15:51:08:052977 gpstop:zlv:gpadmin-[INFO]:-Obtaining Segment details from master...
20200424:15:51:08:052977 gpstop:zlv:gpadmin-[INFO]:-Greenplum Version: 'postgres (Greenplum Database) 7.0.0-alpha.0+dev.6778.g000faed10b build dev'
20200424:15:51:08:052977 gpstop:zlv:gpadmin-[INFO]:-Commencing Master instance shutdown with mode='immediate'
20200424:15:51:08:052977 gpstop:zlv:gpadmin-[INFO]:-Master segment instance directory=/home/gpadmin/workspace/gpdb/gpAux/gpdemo/datadirs/qddir/demoDataDir-1
20200424:15:51:09:052977 gpstop:zlv:gpadmin-[INFO]:-Attempting forceful termination of any leftover master process
20200424:15:51:09:052977 gpstop:zlv:gpadmin-[INFO]:-Terminating processes for segment /home/gpadmin/workspace/gpdb/gpAux/gpdemo/datadirs/qddir/demoDataDir-1
20200424:15:51:09:052977 gpstop:zlv:gpadmin-[INFO]:-Stopping master standby host zlv mode=immediate
20200424:15:51:10:052977 gpstop:zlv:gpadmin-[INFO]:-Successfully shutdown standby process on zlv
20200424:15:51:10:052977 gpstop:zlv:gpadmin-[INFO]:-Targeting dbid [2, 5, 3, 6, 4, 7] for shutdown
20200424:15:51:10:052977 gpstop:zlv:gpadmin-[INFO]:-Commencing parallel primary segment instance shutdown, please wait...
20200424:15:51:10:052977 gpstop:zlv:gpadmin-[INFO]:-0.00% of jobs completed
20200424:15:51:11:052977 gpstop:zlv:gpadmin-[INFO]:-100.00% of jobs completed
20200424:15:51:11:052977 gpstop:zlv:gpadmin-[INFO]:-Commencing parallel mirror segment instance shutdown, please wait...
20200424:15:51:11:052977 gpstop:zlv:gpadmin-[INFO]:-0.00% of jobs completed
20200424:15:51:12:052977 gpstop:zlv:gpadmin-[INFO]:-100.00% of jobs completed
20200424:15:51:12:052977 gpstop:zlv:gpadmin-[INFO]:-----------------------------------------------------
20200424:15:51:12:052977 gpstop:zlv:gpadmin-[INFO]:-   Segments stopped successfully      = 6
20200424:15:51:12:052977 gpstop:zlv:gpadmin-[INFO]:-   Segments with errors during stop   = 0
20200424:15:51:12:052977 gpstop:zlv:gpadmin-[INFO]:-----------------------------------------------------
20200424:15:51:12:052977 gpstop:zlv:gpadmin-[INFO]:-Successfully shutdown 6 of 6 segment instances 
20200424:15:51:12:052977 gpstop:zlv:gpadmin-[INFO]:-Database successfully shutdown with no errors reported
20200424:15:51:12:052977 gpstop:zlv:gpadmin-[INFO]:-Cleaning up leftover shared memory
20200424:15:51:13:052977 gpstop:zlv:gpadmin-[INFO]:-Restarting System...

-- end_ignore

1: SHOW gp_enable_global_deadlock_detector;
 gp_enable_global_deadlock_detector 
------------------------------------
 off                                
(1 row)
